frontend/docs/superpowers/specs/2026-04-14-design-system-toolkit-design.md

```
@@ -0,0 +1,361 @@
+
+### 범위 외 (이번 버전)
+
+- Figma API 직접 연동 (향후 확장)
+- AI 기반 컴포넌트 코드 자동 생성 (향후 확장)
+- 간격(spacing), 그림자(shadow) 등 색상 외 토큰의 ESLint 룰 (향후 확장)
+
+---
+
+## 아키텍처
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)